scripts/js/queries.js

```diff
@@ -321,13 +332,13 @@ function formatInfo(data) {
 
   // Show DNSSEC status if applicable
   var dnssecInfo = "";
-  if (dnssecClass !== false) {
+  if (dnssec.color !== "") {
```

Comment:
```suggestion
  if (dnssec.text !== "N/A") {
```

While being logical equivalent, it feels strange to base the decision on the color status.

Reply:
Actually `dnssec.icon` and `dnssec.color` store strings containing CSS classes for the icon shape and icon color. 

No color means no icon and text = N/A. No icon means no color and text = N/A. Text=N/A means no icon and no color.

The 3 tests are 100% equivalent:
- `if (dnssec.color !== "")`;
- `if (dnssec.icon !== "")`;
- `if (dnssec.text !== "N/A")`;